### PR TITLE
Add onboarding miniscreen app

### PR DESCRIPTION
**Pull request by jcapona**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready/Hold | [Ticket](https://pi-top.atlassian.net/browse/<ticket-number>) |

#### Main changes
- Re-add onboarding miniscreen app. The app should only start after the package is upgraded during onboarding, and on every boot until the device is onboarded.

#### Screenshots (feature, test output, profiling, dev tools etc)

[insert screenshots here]

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: pt_os_web_portal/app.py [hunk 1]
```diff
@@ -39,6 +48,26 @@ def __init__(self, test_mode):
...
+            ):
+                environ["PT_MINISCREEN_SYSTEM"] = "1"
+                logger.debug("Setting ENV VAR to use miniscreen as system...")
+
+                logger.info("Starting miniscreen onboarding application")
+                self.miniscreen_onboarding = OnboardingAssistantApp()
+                self.miniscreen_onboarding.start()
+            else:
+                logger.info("Not a pi-top[4] - disabling AP mode")
+                disable_ap_mode()
```
Comment (angusjfw) [inline pt_os_web_portal/app.py:69]: this should be split out into some other logic - otherwise we are disabling ap on every non-onboarding boot

File: debian/pt-os-web-portal.postinst [hunk 2]
```diff
@@ -44,13 +44,27 @@ case "$1" in
...
 
-  ;;
+	LAST_UPDATE_FOLDER="/var/lib/pt-os-web-portal"
+	STATE_FILE="${LAST_UPDATE_FOLDER}/state.cfg"
 
-\
-  abort-upgrade | abort-remove | abort-deconfigure | try-restart | triggered) ;;
+ 	# if upgrading from a previous version...
+	if [ ! -z "$2" ]; then
+		# and the state file doesn't exist (probably upgrading during onboarding)
```
Comment (angusjfw) [inline debian/pt-os-web-portal.postinst:52]: include in the comment that the web portal usually doesn't have a state file unless it's already been run

Comment (angusjfw) [reply debian/pt-os-web-portal.postinst:52]: eg there is an assumption that old versions of web portal don't create a state file before the update step
